src/app/plan/components/Wizard/GeneralForm.tsx

```diff
@@ -14,21 +13,29 @@ interface IProps {
 }
 
 interface IGeneralFormProps
-  extends Pick<IOtherProps, 'isEdit'>,
-    Pick<FormikActions<IFormValues>, 'setFieldTouched'>,
-    Pick<FormikProps<IFormValues>, 'setFieldValue' | 'values'>,
-    Pick<FormikState<IFormValues>, 'touched'> {
+  extends Pick<
+    IOtherProps, 'isEdit'
+    >,
+    Pick<FormikProps<IFormValues>, 
+    | 'handleBlur'
+    | 'handleChange'
+    | 'setFieldTouched' 
```

Comment:
This is fine for this PR, but I think Prettier lays it out differently. @ibolton336 we should figure out a good time to run Prettier on the whole repo (and merge it quick so we don't cause conflicts). @gildub I would recommend installing the Prettier plugin for VS Code and letting it format your code on save (or if you disable that, you can tell it to format a selection).